UIKitDZ/UIKitDZ/PhotoFeed/Cells/CellType.swift

```diff
@@ -0,0 +1,16 @@
+// CellType.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+//////Протокол предоставляет информацию о типе модели
+protocol CellTypeProtocol {
+    var cellType: TableCellsTypes { get }
+}
+
+/// Энам для типов ячеек
+enum TableCellsTypes {
```

Comment:
```suggestion
enum TableCellsTypes {
```
вынеси + дока на кейс